File: pinyougouparent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll=function () {
        return $http.get('../brand/findAll.do')
    }
    this.findPage=function (page,row) {
        return  $http.get('../brand/findPage.do?page='+page+'&row='+row)
    }
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id)
    }
    this.saves=function (entity) {
        return  $http.post("../brand/saves.do",entity)
    }
    this.update=function (entity) {
        return  $http.post("../brand/update.do", $scope.entity)
    }
    this.dele=function (ids) {
        return $http.get('../brand/dele.do?ids=' +ids)
    }
    this.search=function (page,row,searchEntity) {
        return $http.post("../brand/search.do?page="+page+"&row="+row,searchEntity)
    }
    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do')
    }
});

File: pinyougouparent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {
    /**
     * 查询所有品牌
     * @return
     */
    public List<TbBrand> findAll();


    //分页
    public PageResult findPage(int pageNum,int pageSize);
    //添加品牌
    public void add(TbBrand brand);
    //删除品牌
    public void delete(Long[] ids);
    //修改品牌
    public void update(TbBrand brand);
    //根据id查找需要 修改的品牌
    public TbBrand findOne(Long id);
    //分页查询
    public PageResult findPage(TbBrand brand,int pageNum,int pageSize);
    //查询相关品牌
    List<Map> selectOptionList();
}
